src/Hive.Plugins/Loading/PluginLoadContext.cs

```diff
@@ -52,5 +82,33 @@ protected override IntPtr LoadUnmanagedDll(string unmanagedDllName)
 
             return IntPtr.Zero; // cannot load the dll in this ALC
         }
+
+        private bool disposedValue;
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
+            {
+                if (disposing)
+                {
+                    isResolving.Dispose();
+                }
+
+                disposedValue = true;
+            }
+        }
+
+        // ~PluginLoadContext()
```

Comment:
nit: commented code? Why is this removed as well?

Reply:
This is part of the dispose pattern template, but we don't need it because we don't have unmanaged resources.